Update documentation about installing api-gateway on K3D (#1017)
kyma-project/api-gateway: docs/contributor/01-00-installation.md

    ```bash
    make deploy
    ```

> 7. If installating on k3d make sure to update Core DNS to correctly resolve `local.kyma.dev` domain.

Reviewer: ```suggestion
7. If you're using k3d, update Core DNS to correctly resolve the `local.kyma.dev` domain.
```
